File: samples/jafu-reactive-data-r2dbc/src/main/java/com/sample/Application.java
package com.sample;

import org.springframework.fu.jafu.JafuApplication;

import static org.springframework.fu.jafu.Jafu.reactiveWebApplication;

public abstract class Application {

	public static JafuApplication app = reactiveWebApplication(app -> app
				.enable(Configurations.dataConfig)
				.enable(Configurations.webConfig));

	public static void main (String[] args) {
		app.run(args);
	}
}

File: samples/kofu-coroutines-mongodb/src/main/kotlin/org/springframework/fu/sample/coroutines/Application.kt
package org.springframework.fu.sample.coroutines

import org.springframework.fu.kofu.reactiveWebApplication

val app = reactiveWebApplication {
	configurationProperties<SampleProperties>(prefix = "sample")
	enable(dataConfig)
	enable(webConfig)
}

fun main() {
	app.run()
}
